doc/policy/api_changes.md

```diff
@@ -0,0 +1,125 @@
+# Deprecations
+
+## Policy
+
+External C and C++ API deprecations shall be announced (via `TILEDB_DEPRECATED*` annotations) at least two release cycles before removal.
+
+Timeline:
+- release X.y: `TILEDB_DEPRECATED_EXPORT` added to function declaration.
+- release X.y+1: announce function removal
+- release X.y+2: remove function
+
+## Removal Announcements
+
+Function removal shall be announced one release cycle before removal, following at least one release cycle of deprecation.
+
+**Announcements must be included in the release notes for the release preceding the removal.**
+
+### Announcements
+
+- 2.13: All functions deprecated in TileDB <=2.12 will be removed in TileDB 2.15.
+
+## Deprecation Version History
```

Comment:
Probably don't want the policy and history in the same file in the long term, but it's OK for now.